tests/unit/components/sl-calendar-test.js

```diff
@@ -88,7 +88,7 @@ test( 'Lock mode prevents changing state', function( assert ) {
 });
 
 test( 'Clicking a day with a valid content value sends data', function( assert ) {
-    var testDay = 1,
+    let testDay = 1,
```

Comment:
There should be one assignment per line, per https://github.com/softlayer/ember-style-guide/blob/master/javascript.md#variables